sql_generators/glean_usage/templates/dau_reporting_clients_first_seen.metadata.yaml

```diff
@@ -0,0 +1,19 @@
+{{ header_yaml }}
+friendly_name: DAU Reporting Clients Daily
+description: |-
+  A representation of when we saw each `client_id`
+  for the first time based on the dau_reporting ping.
+
+owners:
+  - kik@mozilla.com
+labels:
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_glean_usage
+  task_group: {{ app_name }}
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: true
```

Comment:
Does a view use these fields in the metadata?

Reply:
copy paste mistake. Thanks for pointing it out.